app/models/crime_application.rb

```diff
@@ -126,32 +107,7 @@ def case_details
     @case_details ||= CaseDetailsPresenter.present(self[:case_details])
   end
 
-  private
-
   def applicant
-    client_details.applicant
-  end
-
-  # rubocop:disable Metrics/MethodLength, Metrics/CyclomaticComplexity
-  def format_telephone_number(number)
-    # Remove all spaces
-    formatted_tel = number.gsub(/\s+/, '')
-
-    case formatted_tel
-    when /\A07\d{9}\z/ # Mobile number with exactly 11 digits
-      [5, 9].each { |i| formatted_tel.insert i, ' ' }
-    when /\A\+447\d{9}\z/ # +44 mobile number format without initial 0
-      [3, 8, 12].each { |i| formatted_tel.insert i, ' ' }
-    when /\A\+4407\d{9}\z/ # +44 mobile number format with initial 0
-      [3, 9, 13].each { |i| formatted_tel.insert i, ' ' }
-    when /\A\+44\(0\)7\d{9}\z/ # +44 mobile numberformat with initial 0 in parentheses
-      [3, 7, 12, 16].each { |i| formatted_tel.insert i, ' ' }
-    else
-      formatted_tel = applicant.telephone_number
-    end
-
-    formatted_tel
+    @applicant ||= ApplicantPresenter.present(self[:client_details][:applicant])
```

Comment:
Is there ever situation where `{ "client_details": { "applicant": null } }` that is - there is no applicant available?

Reply:
Nope there should always be an applicant - https://github.com/ministryofjustice/laa-criminal-legal-aid-schemas/blob/main/schemas/1.0/application.json